aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -1,97 +1,214 @@
-/// This module provides an interface for aggregators (version 2).
-/// Only skeleton - for AggregagtorSnapshot - is provided at this time,
-/// to allow transition of usages.
+/// This module provides an interface for aggregators (version 2). Aggregators are
+/// similar to unsigned integers and support addition and subtraction (aborting on
+/// underflow or on overflowing a custom upper limit). The difference from integers
+/// is that aggregators allow to perform both additions and subtractions in parallel
+/// across multiple transactions, enabling parallel execution. For example, if the
+/// first transaction is doing `try_add(X, 1)` for aggregator `X`, and the second is
+/// doing `try_sub(X,3)`, they can be executed in parallel avoiding a read-modify-write
+/// dependency.
+/// However, reading the aggregator value (i.e. calling `read(X)`) is a resource-intensive
+/// operation that also reduced parallelism, and should be avoided as much as possible.
 module aptos_framework::aggregator_v2 {
+    use std::error;
     use std::string::String;
 
+    /// The value of aggregator overflows. Raised by uncoditional add() call
+    const EAGGREGATOR_OVERFLOW: u64 = 1;
+
+    /// The value of aggregator underflows (goes below zero). Raised by uncoditional sub() call
+    const EAGGREGATOR_UNDERFLOW: u64 = 2;
```

Comment:
I think that was from v1 aggregators, to disambiguate.

though I see only 3 there now:

    const ENOT_SUPPORTED: u64 = 3;

Reply:
because rest are consistent:


    /// The value of aggregator overflows. Raised by native code.
    const EAGGREGATOR_OVERFLOW: u64 = 1;

    /// The value of aggregator underflows (goes below zero). Raised by native code.
    const EAGGREGATOR_UNDERFLOW: u64 = 2;

Reply:
ENOT_SUPPORTED is never used I think